rolling-shutter/medley/broker.go

```diff
@@ -0,0 +1,184 @@
+package medley
+
+import (
+	"sync"
+
+	"github.com/rs/zerolog/log"
+)
+
+// `Broker` allows to distribute a value to multiple
```

Comment:
Why the backticks? I think so far I've only seen unquoted names in godoc comments.

Reply:
True, I think this is an automatism of mine, caused by extensive use of `markdown syntax`